### PR TITLE
Fixes position of upload svg on mobile

### LINKED ISSUE #2671: Placing of image upload icon is off on mobile
This was on an iPhone 7 Plus:

img_1441 png

### PR DESCRIPTION
**Pull request by karmatosed**
This works for any 'not large' device.

Fixes #2671

### This was before on an iPhone 7 Plus

img_1441 png

### After

Mobile:

screen shot 2017-09-22 at 09 04 26

Desktop:

screen shot 2017-09-22 at 09 09 39

### CODE REVIEW

File: editor/header/tools/style.scss [hunk 1]
```diff
@@ -106,6 +106,13 @@
 	margin-bottom: 0;
 }
 
+@media ( max-width: $break-large ) {
+	.wp-core-ui .button.wp-block-image__upload-button.button.button-large {
+	  padding: 6px 21px 6px 6px;
```
Comment (aduth) [inline editor/header/tools/style.scss]: Mixed spaces / tabs here.

**Review by aduth — CHANGES_REQUESTED** [review]
Is #1745 the right issue to be closing? I'm not sure what to be testing here.

**Comment by gziolo** [comment]
@karmatosed, it looks like wrong issue is referenced here. Can you provide some testing instructions here?

File: editor/header/tools/style.scss [hunk 2]
```diff
@@ -106,6 +106,13 @@
 	margin-bottom: 0;
 }
 
+@media ( max-width: $break-large ) {
```
Comment (mtias) [inline editor/header/tools/style.scss]: You can use `@include break-large()` here.

**Comment by karmatosed** [comment]
I have updated with the issue it should have referenced, my apologies it was relating to a different issue. #2693 2671

File: editor/header/tools/style.scss (hunk 2 above)
Comment (gziolo) [reply editor/header/tools/style.scss]: Updated to make it mobile first and moved styling to the component's style file.

**Comment by gziolo** [comment]
@mtias can you validate my style changes? Just saying, I'm not an expert in this field :)

**Comment by mtias** [comment]
@gziolo seems good, though you might also have to do for galleries (that's why I was hoping it could be done for https://github.com/WordPress/gutenberg/tree/master/blocks/media-upload-button directly.

**Comment by gziolo** [comment]
It makes sense, I will update 👍

**Comment by gziolo** [comment]
It's a different component `<FormFileUpload />` :) Anyways updated and it syncs nicely between those two mentioned blocks:

screen shot 2017-09-22 at 13 02 13

File: components/form-file-upload/editor.scss [hunk 3]
```diff
@@ -0,0 +1,11 @@
+.components-form-file-upload {
```
Comment (mtias) [inline components/form-file-upload/editor.scss]: The file should be `style.scss` not `editor`; `editor.scss` is used to differentiate editor styles for _blocks_.

Comment (gziolo) [reply components/form-file-upload/editor.scss]: I see. It makes sense, but it's a bit confusing. I assumed that it needs to be editor view as this is where it only applies :)

Comment (mtias) [reply components/form-file-upload/editor.scss]: The editor / style split is only for blocks, as we need to load those on the front end, and we don't want to pull block styles that are meant for the editor.

Comment (gziolo) [reply components/form-file-upload/editor.scss]: Thanks for making it clear :) It's now updated to match the convention.

**Comment by mtias** [comment]
🚢 it

**Pull request merged**